projects/packages/sync/src/class-functions.php

```diff
@@ -71,7 +71,7 @@ public static function get_shortcodes() {
 	public static function sanitize_taxonomy( $taxonomy ) {
 
 		// Lets clone the taxonomy object instead of modifing the global one.
-		$cloned_taxonomy = json_decode( wp_json_encode( $taxonomy ) );
+		$cloned_taxonomy = json_decode( wp_json_encode( $taxonomy, JSON_UNESCAPED_SLASHES ) );
```

Comment:
Another one of these
```suggestion
		$cloned_taxonomy = json_decode( wp_json_encode( $taxonomy, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE ) );
```

Reply:
24d018fdee